source/server/debug_session_properties_restricted_service.h

```diff
@@ -0,0 +1,62 @@
+#ifndef NIDEVICE_RESTRICTED_GRPC_DEBUG_SESSION_PROPERTIES_RESTRICTED_SERVICE
+#define NIDEVICE_RESTRICTED_GRPC_DEBUG_SESSION_PROPERTIES_RESTRICTED_SERVICE
+
+#include <grpcpp/ext/proto_server_reflection_plugin.h>
+#include <grpcpp/grpcpp.h>
+#include <grpcpp/health_check_service_interface.h>
+#include <debugsessionproperties_restricted.grpc.pb.h>
+
+#include "feature_toggles.h"
+#include "shared_library.h"
+#include "syscfg_library_interface.h"
+#include "syscfg_session_handler.h"
+
+namespace nidevice_restricted_grpc {
+
+static const char* kDebugSessionPropertyAccessFailedMessage = "The NI System Configuration API was unable to access the debug session property.";
```

Comment:
This is only used in the `debug_session_properties_restricted_service.cpp`. Would it make more sense to define this there?

Reply:
This is used by the unit test, so I am leaving it in place.